app.js

```diff
@@ -1,6 +1,21 @@
-let count = 0;
-document.querySelector('.my-pic').addEventListener('click', function(){
-    count++;
-    document.getElementById('btn').innerHTML= count;
-    localStorage.setItem('count','count');
+document.addEventListener('DOMContentLoaded', getCount);
+let counter1 = 0;
+let counter2 = 0;
+document.querySelector('.my-pic1').addEventListener('click', function(){ 
+    counter1++;
+    localStorage.setItem('counter1',counter1);
+    document.getElementById('btn1').innerHTML= localStorage.getItem('counter1');
 });
+
+document.querySelector('.my-pic2').addEventListener('click', function(){
+    counter2++;
+    localStorage.setItem('counter2',counter2);
+    document.getElementById('btn2').innerHTML= localStorage.getItem('counter2');

```

Comment:
avoid using innerHTML, check for alternative methods of doing it, innerHTML is a security vulnerability